book/http.md

```diff
@@ -370,11 +412,11 @@ different types to text and to bytes:
 If you see an error about `str` versus `bytes`, it's because you
 forgot to call `encode` or `decode` somewhere.
 
-If you run this, you'll notice that the `send` call returns a number,
-in this case `47`. That tells you how many bytes of data you sent to
-the other computer; if, say, your network connection failed midway
-through sending the data, you might want to know how much you sent
-before the connection failed.
+If you run this in the REPL, you'll notice that the `send` call
```

Comment:
REPL?

Reply:
Initially the conceit was you'd run all of this code in a Python REPL (that is, in an interpreter on the command line) but now that conceit is gone and this sentence is kind of homeless. I think probably this whole paragraph needs to become a footnote now.

Reply:
I didn't see REPL defined earlier in the chapter, so I don' think readers will understand. Just say "run this via Python on the command line"?

Reply:
Ok. Just make sure it's well-tested...